### DISCUSSION TITLE
iOS open local file url in Safari

### DISCUSSION THREAD

**Discussion by Georg632** [ionic-team/capacitor#4079] [Q&A]
Hi,

Problem:
I'm generating an .ics file on the client and save it with capacitor Filesystem.writeFile. When I try to open this file on iOS with ionic FileOpener I see a Popup Screen with the calendar event preview. But the add to calender button is missing. I tried to open the file from the file browser and get the same result. So it's not capacitor related (but strange...)
Now I copied the path from the saved ics file and opened it in Safari - it is working without any problems!

Question:
Can I somehow open a "file:///.."-url from capacitor in Safari? I tried a lot of plugins but none of them are working.
The capacitor Browser Plugin is limited to 'http' and 'https' calls If I understand it correct.

Alternative:
If there is no way I have to generate the file on the webserver and download it with the Browser Plugin.

I hope anyone can help me!
Thanks :)

**Comment by hatsantos** [comment]
@Georg632, Did you found any solution?

I'm facing the similar issue. I have generated de ICS file, with the JSON response from the app rest API.
But now I don't know what I should do. Save the file and then read it again?

I want to add the event to iOS calendar.
